packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,11 +1,14 @@
 import { PaperPlane } from '@strapi/icons';
 
-import { CMReleasesContainer } from './components/CMReleasesContainer';
+import { ReleaseActionModalForm } from './components/ReleaseActionModal';
+import { Panel as ReleasesPanel } from './components/ReleasesPanel';
+// import { addColumnToTableHook } from './components/ReleaseListCell';
```

Comment:
This is for the "Releases" column in the list view, is not implemented yet and I would like to do it in an upcoming PR. I left the code commented here because I think it should not change too much at this file.